Dshop/apps/products_catalogue/templates/products_catalogue/category_detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<style>
```

Comment:
Please move into CSS file and link in head. 
Simple solution would be to create {% block extrahead %} in base.html.

Reply:
done!